hydromt/cli/main.py

```diff
@@ -118,6 +119,12 @@ def print_models(ctx, param, value):
     default=False,
     help="Flag: If provided overwrite existing model files",
 )
+ignore_empty_opt = click.option(
+    "--ignore-empty",
+    is_flag=True,
+    default=False,
```

Comment:
Can we make for export_data that the default behavior is to ignore empty data (default = IGNORE)?

Reply:
sure!